ramalama/common.py

```diff
@@ -293,9 +271,14 @@ def find_in_cdi(devices: list[str]) -> tuple[list[str], list[str]]:
     # Attempts to find a CDI configuration for each device in devices
     # and returns a list of configured devices and a list of
     # unconfigured devices.
-    cdi = load_cdi_config(['/etc/cdi', '/var/run/cdi'])
-    cdi_devices = cdi.get("devices", []) if cdi else []
-    cdi_device_names = [name for cdi_device in cdi_devices if (name := cdi_device.get("name"))]
+    cdi = load_cdi_config(['/var/run/cdi', '/etc/cdi'])
+    try:
+        cdi_devices = cdi.get("devices", []) if cdi else []
+        cdi_device_names = [name for cdi_device in cdi_devices if (name := cdi_device.get("name"))]
+    except (AttributeError, KeyError, TypeError) as e:
+        logger.debug(f"Unable to process CDI configuration: {e}")
```

Comment:
We are hiding the exception here, shouldn't we at least warn about this situation?